### ISSUE TITLE
Users can specify whether to create a new namespace with Helm template

### ISSUE THREAD

**Issue by nwnt** [Azure/draft#446]
**Is your feature request related to a problem? Please describe.**
We're working on integrating Fleet manager with Automated Deployment (#426).  The role we are targeting to assign onto the integrating app is Fleet RBAC Writer.  However, [helm upgrade -i](https://github.com/Azure/draft/blob/3e92a9912fb2a9a0b0c5ee4bea5f57138eee06e3/template/workflows/helm/.github/workflows/azure-kubernetes-service-helm.yml#L134C62-L134C232) command in the draft generated template always creates a namespace, which can't be achieved by the role we want to assign to Fleet as a namespace is a cluster scoped resource.  We do not want to just change the role to Cluster Writer yet.  This is because in Fleet Manager we want to differentiate 2 personas: one for cluster admin app that can create cluster scoped objects; another for application team app that can create ns scoped objects like deployments.

Can we propose that we introduce a new parameter that will make `--create-namespace-` flag in the current helm command run optional?  If so we can start working on a PR.

**Mention what platform you want to support the new feature**
Existing platform.

**Describe the solution you'd like**
Can we propose that we introduce a new parameter that will make `--create-namespace-` flag in the current helm command run optional?  If so we can start working on a PR.

**Describe alternatives you've considered**
Using a cluster writer role, but we don't want to assume that the same app will do both application deployment and cluster management tasks.

**Comment by davidgamero** [comment]
yes adding a createnamespace variable that defaults to true would be in line with the current workflow templates

 you can set that to false when selecting fleet clusters

we should enable this across all deploy types (helm, kustomize, manifests) to present a uniform experience
